### PR TITLE
Fix #416: Add an issue and PR template.

### LINKED ISSUE #416: Consider ISSUE_TEMPLATE.md
https://github.com/blog/2111-issue-and-pull-request-templates

### PR DESCRIPTION
**Pull request by starcraftman**
As discussed in #416. Any other changes/requests? No rush so we can think on it a bit more.

I also tinkered with the README making a get help section under installation, think this is a slightly better place.

### CODE REVIEW

File: .github/ISSUE_TEMPLATE.md [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+### Before Submitting
+
+- You checked the [faq](https://github.com/junegunn/vim-plug/wiki/faq) for common problems.
+- Check your [requirements](https://github.com/junegunn/vim-plug/wiki/requirements) are satisfied.
+- You are not going to suggest vim-plug manage itself like Vundle, see #240, #364, #367 ...
+-->
+
+Explain the problem here ...
+
+### Category Information
```
Comment (starcraftman) [inline .github/ISSUE_TEMPLATE.md]: Maybe just a separator here? Like
----------------------

Comment (junegunn) [inline .github/ISSUE_TEMPLATE.md]: Yeah, it looks better.

File: README.md [hunk 2]
```diff
@@ -49,6 +49,15 @@ $uri = 'https://raw.githubusercontent.com/junegunn/vim-plug/master/plug.vim'
 (New-Object Net.WebClient).DownloadFile($uri, $ExecutionContext.SessionState.Path.GetUnresolvedProviderPathFromPSPath("~\vimfiles\autoload\plug.vim"))
 ```
 
+### Getting Help
+
+- See the [requirements] page for debugging information & tested configurations.
+- See the [FAQ] for common problems and questions.
+- Make an issue.
```
Comment (junegunn) [inline README.md]: A minor nitpick. ["Create an issue"](https://help.github.com/articles/creating-an-issue/) sounds more familiar to me.

**Comment by starcraftman** [comment]
@junegunn Anything else?

**Comment by junegunn** [comment]
How about we ask for the out put of `:version` command? It's not always relevant, but it'll remove unnecessary ping-pongs in many cases.

**Comment by starcraftman** [comment]
@junegunn Done. Hmmm, anything else? Let's think.

File: .github/ISSUE_TEMPLATE.md [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+- Check your [requirements](https://github.com/junegunn/vim-plug/wiki/requirements) are satisfied.
+- You are not going to suggest vim-plug manage itself like Vundle, see #240, #364, #367 ...
+-->
+
+Explain the problem here ...
+
+------------------------------
+
+<!-- Put the contents of `:version` below -->
+```viml
```
Comment (junegunn) [inline .github/ISSUE_TEMPLATE.md]: `viml` seems unnecessary.

**Comment by junegunn** [comment]
I think it's good to go. We can improve it over time.

**Comment by junegunn** [comment]
Thank you!

**Pull request merged**